### ISSUE TITLE
A Post-100% Completion World

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1165]
As we near 100% completion in a city, I'm prompted to think about how to:

- Manage this data over time. How do we keep it up-to-date? Should new volunteers/turkers be routed not just along audit paths that have low confidence but also those that have new panos, for example? How can users submit their own reports/pictures of out-of-date areas?
- How can we track the accessibility of an area over time as we re-audit areas from one year to the next.

**Comment by misaugstad** [comment]
Still thinking about the recency of GSV imagery and how we can use checking for expired panos as a way to prioritize routing. And I'm mostly commenting to add a few more keywords so that it is easier for me to search for this issue in the future :)

**Comment by jonfroehlich** [comment]
Yes, I think using this info to prioritize re-audits is a great idea!

**Comment by misaugstad** [comment]
I am currently working on including the age of imagery for all panos visited in our database, instead of just the ones with labels. This will help us determine the age of imagery on streets more comprehensively.

When prioritizing reaudits, we may also want to take into account labels that were added with the 'construction' tag that have new imagery. Those should probably be prioritized above anything else!

**Comment by misaugstad** [comment]
Closing — the concrete asks here have all shipped or moved to live follow-ups.

The data-freshness machinery got built in 2026 under #4384: per-street imagery age is captured and stored (`street_imagery`, #4347/#4348 → PRs #4352/#4358), routing now sends labelers back to streets with newer imagery and treats "up to date" as the completion bar (PR #4649), re-audit need is surfaced in metrics, the v3 API, admin, and LabelMap (PR #4650), and a nightly poller keeps imagery age current for both GSV and Mapillary cities (PR #4651).

The rest of the thread has its own issues:
- Tracking a place's accessibility from one year to the next → #4792 (per-imagery-era exploration metrics).
- Mikey's 2022 point about prioritizing `construction`-tagged labels once new imagery lands → #4894.
- Users reporting out-of-date/missing imagery → #4922 (shipped) and #4411.
- Telling a labeler they're re-auditing → #4895; what happens to their old-imagery labels → #4945; showing it in Gallery → #4652.
